css/style.css

```diff
@@ -346,3 +360,92 @@ ul li a {
   padding: 12px;
   margin: 0;
 }
+
+.aboutme-text {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  padding: 20px;
+}
+
+form {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  gap: 20px;
+  background-image: url(/images/illustration-contact-form.png);
```

Comment:
**OPTIONAL** - Please follow the advice of the previous reviewer in navigating to your images folder, your background image is still not visible. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. 
`notice how the image for medium icon is broken and also the background image is not showing up, as recommended earlier correct path for your medium icon = <img src="./images/iconmonstr-medium-1.svg">
correct path for you background image =background-image: URL(../images/illustration-contact-form.svg);`

Your project:
<img width="198" alt="efef" src="https://user-images.githubusercontent.com/73607512/213139274-5d1fe6a6-cf15-468c-bdf9-44252394a20f.PNG">